tensilelite/Tensile/Tensile.py

```diff
@@ -101,7 +121,16 @@ def executeStepsInConfig(
                 libraryLogicConfig = config["LibraryLogic"]
             else:
                 libraryLogicConfig = {}
-            LibraryLogic.main(libraryLogicConfig, srcToolchain.compiler, outputPath)
+            LibraryLogic.main(
+                libraryLogicConfig, 
+                srcToolchain.compiler, 
```

Comment:
should this be a string?
```suggestion
                str(srcToolchain.compiler.path) 
```